.travis.yml

```diff
@@ -266,7 +269,7 @@ script:
   # Go to the python wrapping package directory.
   - if [ "$WRAP" = "on" ]; then cd $OPENSIM_HOME/sdk/Python; fi
   # Run the python tests, verbosely.
-  - if [ "$WRAP" = "on" ]; then /usr/bin/python -m unittest discover --start-directory opensim/tests --verbose; fi
+  - if [ "$WRAP" = "on" ]; then /usr/bin/python3 -m unittest discover --start-directory opensim/tests --verbose; fi
```

Comment:
This probably works only on Linux. We can probably change this to just `python3` now (remove `/usr/bin`, which was there for Mac).

Reply:
Done. Will see if that fixes the osx build without breaking the linux build. Thanks @chrisdembia